grakn-engine/src/main/java/ai/grakn/engine/tasks/manager/ZookeeperConnection.java

```diff
@@ -84,7 +83,7 @@ public CuratorFramework connection(){
         return zookeeperConnection;
     }
 
-    public InterProcessMutex mutex(TaskId id){
+    public InterProcessMutex mutex(String id){
```

Comment:
You can now get a mutex on the engine ID or the task ID. This is why I was asking about that ID interface earlier ;) Related: in `TaskId` we use `getValue` and in `EngineID` we use `value()`